packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -28,16 +28,22 @@ interface MutationNameMap {
     delete?: string;
 }
 
+enum ModelSubscriptionLevel {
```

Comment:
Duplicate from ```AuthTransformer```, can't we share these types somehow? It is also missing the ```ON``` value, see my comment above at ```AuthTransformer```